src/include/migraphx/op/prefix_scan_op.hpp

```diff
@@ -21,6 +21,13 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+
+/**
+ * Parent struct for prefix scan ops.  A prefix scan is a mathematical entity useful
+ * in parallelizing various computations.  Given a list of numbers, a prefix scan
```

Comment:
Its not just useful in parallel contexts. I think it would be simpler to just say that prefix scan is equivalent to `std::exclusive_scan` or `std::inclusive_scan`.